Assets/Scripts/UIScripts/PauseScreen.cs

```diff
@@ -1,3 +1,5 @@
+using System.Collections;
+using System.Collections.Generic;
```

Comment:
Minorrrrrrrrrrrrrrr comment: My IDE tells me "using System.Collections.Generic;" is unused